rise/static/main.js

```diff
@@ -387,41 +403,31 @@ function buttonHelp() {
     var help_button = $('<i/>')
         .attr('id','help_b')
         .attr('title','Reveal Shortcuts Help')
-        .addClass('fa-question fa-4x fa')
+        .addClass('fa-question fa-2x fa')
         .addClass('my-main-tool-bar')
-        .css('position','fixed')
```

Comment:
I've moved these properties to the CSS. LMK if I should revert this and the following hunks back.